app/models/git_hub_team.rb

```diff
@@ -0,0 +1,20 @@
+class GitHubTeam
+  attr_reader :id, :name
+
+  def initialize(github_client, org_id, id=nil, name=nil)
+    @github_client = github_client
+    @org_id        = org_id
+    @id            = id
+    @name          = name
+  end
+
+  def find_or_create_team(team_id, team_name)
```

Comment:
Should this be a class method since it returns a `GitHubTeam` object?

Maybe `GitHubTeam.find_or_create(team_id, team_name)`